requirements/edx/base.txt

```diff
@@ -156,3 +156,6 @@ analytics-python==0.4.4
 # Needed for mailchimp(mailing djangoapp)
 mailsnake==1.6.2
 jsonfield==1.0.3
+
+# Needed for credit notifications
+pynliner==0.5.2
```

Comment:
@wedaly can you add the context from the FYI into the comment.  Also can you tell my why we are doing this instead of just embedding the css string into a `<style>` tag in the HTML?  If there is a good reason for that, should this be rolled out into more of our e-mails?  If so we should create ticket for aligning all our e-mail tooling to use it.